packages/apps/policy-send-counter/src/lib/schemas.ts

```diff
@@ -1,71 +1,104 @@
 import { z } from 'zod';
 
+/**
+ * Ability parameters schema - matches the ability this policy works with
+ */
 export const abilityParamsSchema = z.object({
-  ethRpcUrl: z
+  to: z
     .string()
-    .describe(
-      'An Ethereum Mainnet RPC Endpoint. This is used to check USD <> ETH prices via Chainlink.',
-    ),
-  rpcUrlForUniswap: z
+    .min(1, 'Recipient address cannot be empty')
```

Comment:
Does this error if you attempt to send to `0x0000000000000000000000000000000000000000` for burning?